homeassistant/components/somfy/__init__.py

```diff
@@ -60,17 +64,21 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     hass.data[DOMAIN][CONF_OPTIMISTIC] = domain_config.get(CONF_OPTIMISTIC, False)
 
     if CONF_CLIENT_ID in domain_config:
-        config_flow.SomfyFlowHandler.async_register_implementation(
+        await async_import_client_credential(
             hass,
-            config_entry_oauth2_flow.LocalOAuth2Implementation(
-                hass,
-                DOMAIN,
+            DOMAIN,
+            ClientCredential(
                 config[DOMAIN][CONF_CLIENT_ID],
                 config[DOMAIN][CONF_CLIENT_SECRET],
-                "https://accounts.somfy.com/oauth/oauth/v2/auth",
-                "https://accounts.somfy.com/oauth/oauth/v2/token",
             ),
         )
+        _LOGGER.warning(
+            "Configuration of Somfy integration OAuth credentails in YAML is "
```

Comment:
The Somfy integration is already deprecated and scheduled for removal in July. Maybe we should just leave it as is and not change anything?

Reply:
Indeed, it can confuse end users.

Reply:
Ah! yes, thank you, i'll close this and double check that before the few integrations I convert.